README.md

```diff
@@ -6,39 +6,111 @@ Please edit this README.md file with your answers to the worksheet questions.
 ### b
 
+|    A    |    B    |  NOT A  |   NOT B |  NOT(A AND B) |   NOT A OR NOT B |
+|---------|---------|---------|---------|---------------|------------------|
+|    0    |    0    |    1    |    1    |    1          |    1             |
+|    1    |    0    |    0    |    1    |    1          |    1             |
+|    0    |    1    |    1    |    0    |    1          |    1             |
+|    1    |    1    |    0    |    0    |    0          |    0             |
+
 ### c
 
+|    A    |    B    |    C    |    A AND B    |   A AND C    |   B OR C |   (A AND B) OR (A AND C)    |    A AND (B OR C)   |
+|---------|---------|---------|--------------|-------------|-----------|---------------------|---------------|
+|    0    |    0    |    0    |    0         |    0        |    0      |    0                |    0          |
+|    1    |    1    |    1    |    1         |    1        |    1      |    1                |    1          |
+|    1    |    1    |    0    |    1         |    0        |    1      |    1                |    1          |
+|    1    |    0    |    0    |    0         |    0        |    0      |    0                |    0          |
+|    0    |    1    |    1    |    0         |    0        |    1      |    0                |    0          |
+|    0    |    0    |    1    |    0         |    0        |    1      |    0                |    0          |
+|    1    |    0    |    1    |    0         |    1        |    1      |    1                |    1          |
+|    0    |    1    |    0    |    0         |    0        |    1      |    0                |    0          |
+
 ### d
 
+|    A    |    B    |    C    |   A OR B |   A OR C |   B AND C |  (A OR B) AND (A OR C)    |   A OR (B AND C) |
+|---------|---------|---------|----------|----------|-----------|---------------------------|------------------|
+|    0    |    0    |    0    |    0     |    0     |    0      |    0                      |    0             |
+|    1    |    1    |    1    |    1     |    1     |    1      |    1                      |    1             |
+|    1    |    1    |    0    |    1     |    1     |    0      |    1                      |    1             |
+|    1    |    0    |    0    |    1     |    1     |    0      |    1                      |    1             |
+|    0    |    1    |    1    |    1     |    1     |    1      |    1                      |    1             |
+|    0    |    0    |    1    |    0     |    1     |    0      |    0                      |    0             |
+|    1    |    0    |    1    |    1     |    1     |    0      |    1                      |    1             |
+|    0    |    1    |    0    |    0     |    0     |    0      |    0                      |    0             |
+
 ## Question 4
 
 ### a
-
+From DeMorgan’s Law, not (a.b) = not a or not b
 ### b
-
+DeMorgan’s Law, x(int) and x>7 or x (float) and x > 7 == x(int) or x(float) and x > 7
```

Comment:
Correct reasoning, but this isn't de Morgan's law (it's the distributive law -- also equivalent to question 3c)